### ISSUE TITLE
Royal Jelly

### ISSUE THREAD

**Issue by cyphun24** [EdenServer/community#592]
the mobs are not casting spells. and the chest doesnt spawn sometimes

**Comment by 04ayresl** [comment]
Also lost Star orbs due to no chest spawn and GM had to come warp me out

**Comment by godmode** [comment]
Did you guys happen to exit the bcnm on some characters before one person opened the chest? This was a common retail strategy

**Comment by 04ayresl** [comment]
i dont even know what that means, both my characters were still in the bcnm when Jailer (GM) come to get me out, the bcnm Server died 10 mins before that aswell

**Comment by 04ayresl** [comment]
will we be able to get out orbs back Godmode or are they lost into the void?

**Comment by cyphun24** [comment]
after it died no chest spawned at all.

**Comment by Setz0r** [comment]
did the queen spawn or did you kill all the jellies before it could?

**Comment by baroom84** [comment]
No one exited, we waited until servers went down with no chest.

**Comment by baroom84** [comment]
We didnt kill all the slimes, queen spawned we killed queen and no chest popped.

**Comment by 04ayresl** [comment]
same as killed Queen and no chest, GM was in the zone with me as well as i /yelled when it happened, so he saw there was no mob or chest and he had to warp me out

**Comment by godmode** [comment]
Yes, orbs lost due to server error will be restored. Just take screenshots

**Comment by 04ayresl** [comment]
i dont have a screenshot and already warped but GM Jailer was in the zone with me and had to warp me out.... dont know if that counts as evidence

**Comment by baroom84** [comment]
![baroom_2018 12 26_171504](https://user-images.githubusercontent.com/46089689/50460047-d1441380-093f-11e9-8296-4efe813e378f.png)

**Comment by Purplekjw** [comment]
![jardir_2019 01 26_162401](https://user-images.githubusercontent.com/16762604/51790255-78f47180-218a-11e9-8c4c-ce6b122d7b1e.png)

Happened a short while ago, no chest appeared. We believe the reason to be that another group was already doing the RJ when we entered the instance. (We didn't realize until afterwards)

Was hoping to get my seals refunded. Thanks!

**Comment by godmode** [comment]
Please make a GM call addressed to godmode with the orb lost
